src/backend/.docker/Dockerfile.local

```diff
@@ -1,25 +1,25 @@
-FROM node:18.9-alpine
-
-RUN apk update
-
-# Installs latest Chromium (100) package.
-RUN apk add --no-cache \
-      chromium \
-      nss \
-      freetype \
-      harfbuzz \
-      ca-certificates \
-      ttf-freefont
+FROM node:20.5-bullseye-slim
+
+# Update packages and install the required dependencies
+RUN apt-get update && \
+    apt-get install -y \
+    chromium \
+    freetype2-demos \
+    libharfbuzz0b \
+    ca-certificates \
+    fonts-freefont-ttf && \
+    apt-get clean && \
```

Comment:
any benefit from cleaning and removing apt-get? since this is local would this be necessary?

Reply:
It's best practice and this is replacing the previous functionality of `--no-cache`